74. Search a 2D Matrix.py

```diff
@@ -0,0 +1,34 @@
+# n = len(nums)
+# time: O(n)
```

Comment:
You must write an algorithm with O(log n) runtime complexity.
#n = row_num * col_num